torch/testing/_internal/common_utils.py

```diff
@@ -1987,22 +1989,26 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, hig
         complex type and 1 if integer/boolean type.
     """
 
-    assert low is None or low < 9, "low value too high!"
-    assert high is None or high > -9, "high value too low!"
+    if low is not None and high is not None:
+        assert low <= high, "Expected low value to be lower than or equal to high, but found higher instead."
+        assert high >= low, "Expected high value to be higher than or equal to low, but found lower instead."
 
     if dtype is torch.bool:
         result = torch.randint(0, 2, size, device=device, dtype=dtype)
     elif dtype is torch.uint8:
-        low = math.floor(0 if low is None else max(low, 0))
-        high = math.ceil(10 if high is None else min(high, 10))
+        ranges = (torch.iinfo(dtype).min, torch.iinfo(dtype).max)
+        low = math.floor(ranges[0] if low is float('-inf') else 0 if low is None or low < ranges[0] else low)
+        high = math.ceil(ranges[1] if high is float('inf') else 10 if high is None or high >= ranges[1] else high)
```

Comment:
This is quite hard to read. I suggest you either put some parentheses around the second `if / else` or better yet write it as regular `if / elif`.